wagtail/contrib/wagtailfrontendcache/utils.py

```diff
@@ -56,18 +56,84 @@ def get_backends(backend_settings=None, backends=None):
 
 
 def purge_url_from_cache(url, backend_settings=None, backends=None):
-    for backend_name, backend in get_backends(backend_settings=backend_settings, backends=backends).items():
+    for backend_name, backend in get_backends(backend_settings, backends).items():
         logger.info("[%s] Purging URL: %s", backend_name, url)
         backend.purge(url)
 
 
-def purge_page_from_cache(page, backend_settings=None, backends=None):
+def purge_urls_from_cache(urls, backend_settings=None, backends=None):
+    for backend_name, backend in get_backends(backend_settings, backends).items():
+        for url in urls:
+            logger.info("[%s] Purging URL: %s", backend_name, url)
+
+        backend.purge_batch(urls)
+
+
+def _get_page_cached_urls(page):
     page_url = page.full_url
     if page_url is None:  # nothing to be done if the page has no routable URL
-        return
+        return []
+
+    return [
+        page_url + path[1:]
```

Comment:
Just an observation: I know that this duplicates [the existing logic](https://github.com/wagtail/wagtail/blob/master/wagtail/contrib/wagtailfrontendcache/utils.py#L72), but I wonder if using `path[1:]` as a way of stripping the leading slash is a bit of a blunt tool as opposed to something like `path.lstrip('/')`.  [The docs demonstrate](http://docs.wagtail.io/en/stable/reference/contrib/frontendcache.html#invalidating-more-than-one-url-per-page) using a trailing slash but I can imagine a developer omitting it by mistake, thus leading to hard-to-debug errors where the first character of the subpath is cut off.

Reply:
Ahh yes, I hadn't thought of that. Will implement with ``.lstrip('/')`` instead